src/hotspot/share/opto/memnode.cpp

```diff
@@ -2801,7 +2827,35 @@ Node* StoreNode::Identity(PhaseGVN* phase) {
       val->in(MemNode::Address)->eqv_uncast(adr) &&
       val->in(MemNode::Memory )->eqv_uncast(mem) &&
       val->as_Load()->store_Opcode() == Opcode()) {
-    result = mem;
+        // Handle StoreVector with offsets and masks
```

Comment:
Also: the indendation is not right: it should only be 2 spaces from the `if`.

Reply:
Yep, fixed, thanks.